controllers/networkfence_controller.go

```diff
@@ -18,38 +18,135 @@ package controllers
 
 import (
 	"context"
+	"errors"
+	"fmt"
 
+	conn "github.com/csi-addons/kubernetes-csi-addons/internal/connection"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/proto"
+	"github.com/csi-addons/kubernetes-csi-addons/internal/util"
```

Comment:
move local imports to the last section in this block

Reply:
this still needs to be done, at line 38 this repository is already referenced. Move the new imports of this repository to the last section where `csiaddonsv1alpha1` is imported.

Reply:
Updated now :+1: